src/main/kotlin/graphics/scenery/controls/SwingBridgeFrame.kt

```diff
@@ -0,0 +1,176 @@
+package graphics.scenery.controls
+
+
+import graphics.scenery.utils.Image
+import graphics.scenery.utils.SystemHelpers.Companion.logger
```

Comment:
Wrong import, if you want to use a logger, declare a `private val logger by LazyLogger()`